sdk/messaging/eventgrid/azsystemevents/models.go

```diff
@@ -244,6 +270,15 @@ type APIManagementUserUpdatedEventData struct {
 	ResourceURI *string
 }
 
+// APISpecificationDetails - API specification details.
+type APISpecificationDetails struct {
+	// Specification name.
+	Name *string
+
+	// Specification version.
+	Version *string
+}
+
 // AcsChatEventBaseProperties - Schema of common properties of all chat events
 type AcsChatEventBaseProperties struct {
```

Comment:
Can we capitalize `Acs`?

Reply:
Fixed! Also, got the bonus acronym of Iot capitalized.